lib/auth/grpcserver.go

```diff
@@ -40,6 +40,7 @@ import (
 	"github.com/golang/protobuf/ptypes/empty"
 	"github.com/gravitational/trace"
 	"github.com/gravitational/trace/trail"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
I know this was badly formatted before this PR but remember to divide imports into std, gravitational and other.